### DISCUSSION TITLE
What are the meanings of different "admission type" values in MIMIC IV?

### DISCUSSION THREAD

**Discussion by OmarMHamed** [MIT-LCP/mimic-code#1215] [MIMIC-IV] [answered]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
Hello,
I have a question regarding the different values of admission_type in admission table, and how can to find out if an admission is an urgent admission or an elective (planned) one?

The admission type contains the following values:
‘AMBULATORY OBSERVATION’, 
‘DIRECT EMER.’, 
‘DIRECT OBSERVATION’, 
‘ELECTIVE’, 
‘EU OBSERVATION’, 
‘EW EMER.’, 
‘OBSERVATION ADMIT’, 
‘SURGICAL SAME DAY ADMISSION’, 
‘URGENT’

Thanks,

**Comment by MickeyPark** [comment]
I got a same question. Could you share the review after getting answer?

**Comment by mirkompcr** [comment, marked as answer]
I am a Swiss physician somewhat familiar with the US health care system. That's how I would address this:

Mapping: 
-- Emergency: ‘AMBULATORY OBSERVATION’, ‘DIRECT EMER.’, ‘URGENT’, ‘EW EMER.’, ‘DIRECT OBSERVATION’, ‘EU OBSERVATION’, ‘OBSERVATION ADMIT’
-- Elective: ‘ELECTIVE’, ‘SURGICAL SAME DAY ADMISSION’

**Reply by marymlucas** [reply]
I agree with @mirkompcr, this is how I would group them as well.

**Comment by masilenccee** [comment]
The classification of admissions types is also unfamiliar to me. I have decided to adopt the mapping scheme provided by mirkompcr in my paper! Thanks very much!@mirkompcr
